drivers/bme680/include/bme680_params.h

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright (C) 2019 Mesotic SAS
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     drivers_bme680
+ *
+ * @{
+ * @file
+ * @brief       Default configuration for BME680 device driver
+ *
+ * @author      Dylan Laduranty <dylan.laduranty@mesotic.com>
+ */
+
+#ifndef BME680_PARAMS_H
+#define BME680_PARAMS_H
+
+#include "board.h"
+#include "bme680.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @name    Set default configuration parameters for the BME680
+ * @{
+ */
+
+#ifdef MODULE_PERIPH_I2C
+#ifndef BMX280_PARAM_I2C_DEV
+#define BMX280_PARAM_I2C_DEV         I2C_DEV(0)
+#endif
+#ifndef BMX280_PARAM_I2C_ADDR
+#define BMX280_PARAM_I2C_ADDR        (0x77)
+#endif
+#endif /* MODULE_PERIPH_I2C */
+
+#ifdef MODULE_PERIPH_SPI
+
+#endif
+
+/* Defaults Parameters if none provided */
+#define BME680_PARAMS_DEFAULT                  \
```

Comment:
```suggestion
#define BME680_PARAMS                  \
```
See the comment below.

Reply:
Renamed